Travis is failing on xwiki-platform validation

build failure in [Travis](https://travis-ci.org/github/checkstyle/checkstyle/jobs/769177295#L6970)
```
[INFO] --- maven-checkstyle-plugin:3.1.2:check (default) @ xwiki-platform-test-integration ---

[INFO] Starting audit...

[ERROR] /home/travis/build/checkstyle/checkstyle/.ci-temp/
xwiki-platform/xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-integration/
src/main/java/org/xwiki/test/integration/maven/ArtifactResolver.java:63:39: 
The String "jar" appears 2 times in the file. [MultipleStringLiterals]

[Audit done.

[INFO] There is 1 error reported by Checkstyle 8.43-SNAPSHOT with checkstyle.xml ruleset.

[ERROR] src/main/java/org/xwiki/test/integration/maven/ArtifactResolver.java:[63,39] (coding) 
MultipleStringLiterals: The String "jar" appears 2 times in the file.
```

--- a/.ci/travis/travis.sh
+++ b/.ci/travis/travis.sh
@@ -313,7 +313,6 @@ no-error-xwiki)
   cd ..
   checkout_from https://github.com/xwiki/xwiki-platform.git
   cd .ci-temp/xwiki-platform
-  git checkout "1d243e9a31589c7a2b2b730bd5e""ad67cb989ca8b"
   # Validate xwiki-platform
   mvn -e --no-transfer-progress checkstyle:check@default -Dcheckstyle.version=${CS_POM_VERSION}
   cd ..

